NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/ToolbarView.java

```diff
@@ -131,6 +133,12 @@ private void setTintColor(Drawable icon) {
             else
                 icon.clearColorFilter();
         }
+        if (collapseSearchButton != null) {
+            if (tintColor != null)
+                collapseSearchButton.setColorFilter(tintColor);
+            else
+                collapseSearchButton.clearColorFilter();
+        }
```

Comment:
Thanks, the code looks so good. One last thing I hope!

Can you move this code into its own `setCollapseSearchTintColor` because it's called from two separate places.
Also, please can you call it from the `setTintColor(Integer tintColor)` method instead of the `setTintColor(Drawable icon)` method. **Make sure you merge in the latest master before you make this change because I [edited this setTint method when I tinted the menu items the other day](https://github.com/grahammendick/navigation/pull/400/files#diff-8993ef8d38c6f978f1483270fcaa06c5R125)**.. 
```java
void setTintColor(Integer tintColor) {
    this.tintColor = tintColor;
    setTintColor(getNavigationIcon());
    setTintColor(getLogo());
    setTintColor(getOverflowIcon());
    setCollapseSearchTintColor()
    ...
}
```